doc/new.md

```diff
@@ -0,0 +1,114 @@
+# Welcome to sourmash!
+
+```{contents} Contents
+:depth: 3
+```
+
+sourmash is a command-line tool and Python/Rust library for
+**metagenome analysis** and **genome comparison** with k-mers.  It
+supports the compositional analysis of metagenomes, rapid search of
+large sequence databases, and flexible taxonomic analysis with both
+NCBI and GTDB taxonomies. sourmash works well with sequences 30kb or
+larger, including bacterial and viral genomes.
+
+You might try sourmash if you want to -
+
+* identify which reference genomes to map your metagenomic reads to
```

Comment:
```suggestion
* identify which reference genomes map to your metagenomic reads
```

Reply:
typically, you map reads to reference genomes, not vice versa. So I agree the English isn't necessarily great but the order of things is correct ;)